IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcFaceExcavation/DocEntity.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcFaceExcavation" UniqueId="1efd49fa-50d1-493a-a0b9-a0f50ea38666" DiagramNumber="1" BaseDefinition="IfcUndergroundExcavation" EntityFlags="32">
```

Comment:
Propose renaming to `IfcUndergroundFaceExcavation`.

Reply:
Or as UndergroundExcavation Type

Reply:
Not sure about this proposal. @larswik ?

Reply:
I think that we should go for IfcUndergroundFaceExcavation. Having a type doesn't work since face-/radial excavations are separate subtypes of underground excavation with their own sets of predefined types.

Reply:
I am now a bit confused. I misunderstood the request of @JontyDunn that you now explained. Do we now just leave as is or rename?

Reply:
As a standalone entity, I think that it becomes clearer when renaming. However, it is a subtype of IfcUndergroundExcavation so...?